towhee/functional/data_collection.py

```diff
@@ -727,6 +728,21 @@ def inner():
 
         return self.factory(inner())
 
+    def head(self, n: int = 5):
```

Comment:
please add some code example for user API

Reply:
> 

added a simple example